.babelrc

```diff
@@ -1,10 +1,24 @@
 {
   "presets": [
     "stage-2",
-    "react",
     ["env", {
+      "targets": {
+        "browsers": ["last 2 versions"]
+      },
       "modules": false
-    }]
+    }],
+    "react"
   ],
+  "env": {
+    "test": {
+      "presets": [
+        ["env", {
+          "targets": {
+            "node": "current"
+          }
+        }]
+      ]
+    }
+  },
   "plugins": ["transform-class-properties"]
```

Comment:
stage-2 includes this plugin

Reply:
without this line tests and build are failing. If I move this line to `"env": { "test": {  } }`. Test are passing but build is failing